src/mongodb/database.py

```diff
@@ -87,7 +87,7 @@ def __init__(
         user: str = "root",
         password: str = "password",
         port: int = 27017,
-        connection_opts: str = "?authSource=admin",
+        # connection_opts: str = "?authSource=admin",
```

Comment:
Behold dette alternativet
```suggestion
        connection_opts: str = "?authSource=admin",
```